session/manager.go

```diff
@@ -0,0 +1,33 @@
+package session
+
+type Manager struct {
+	Sessions []SessionId
+}
+
+func (manager *Manager) Create() (SessionId, error) {
+	sid, err := GenerateSessionId()
+	if err != nil {
+		return sid, err
+	}
+
+	for !manager.sessionIsUnique(sid) {
```

Comment:
What if SID will have collision with another node's session?
Maybe let add ticket for future improvement